xml/System.Net.WebSockets/ClientWebSocket.xml

```diff
@@ -369,7 +369,17 @@
       <Docs>
         <param name="uri">The URI of the WebSocket server to connect to.</param>
         <param name="cancellationToken">A cancellation token used to propagate notification that the operation should be canceled.</param>
-        <summary>Connects to a WebSocket server as an asynchronous operation.</summary>
+        <summary>
+			Connects to a WebSocket server as an asynchronous operation.
```

Comment:
The resulted page will not retain this formatting, you can see it on the preview link generated https://review.learn.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocket.connectasync?view=net-7.0&branch=pr-en-us-8494#system-net-websockets-clientwebsocket-connectasync(system-uri-system-threading-cancellationtoken). I think this information might look better in the `Remarks` section, which supports formatting.

Reply:
Moved to remarks